packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -0,0 +1,37 @@
+import {Component, Element, h} from '@stencil/core';
+import ArrowRightIcon from 'coveo-styleguide/resources/icons/svg/arrow-right-rounded.svg';
+import ArrowLeftIcon from 'coveo-styleguide/resources/icons/svg/arrow-left-rounded.svg';
+import {Button} from '../../common/button';
+
+@Component({
+  tag: 'atomic-segmented-facet-scrollable',
+  styleUrl: 'atomic-segmented-facet-scrollable.pcss',
+  shadow: true,
+})
+export class AtomicSegmentedFacetScrollable {
+  @Element() private host!: HTMLElement;
```

Comment:
unused
```suggestion
```

Reply:
it will be used to add functionality for the arrows